src/index.html

```diff
@@ -11,8 +12,66 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <!--#google Font-->
+    <!--crossorigin-->
+    <link
+      rel="stylesheet"
+      href="./style.scss"
+    />
```

Comment:
The link tag is referencing a SCSS file (`./style.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS into CSS and link to the resulting CSS file instead.